### PR TITLE
ingress: Allow removal of the ingress TLS section

### PR DESCRIPTION
**Pull request by adinhodovic**
Allows disabling TLS if it's externally terminated (LB). Currently it's always defined, at least the hosts section.

https://github.com/pomerium/pomerium-helm/blob/main/charts/pomerium/templates/_helpers.tpl#L621-L641 (picks up ingress.hosts which I want defined).

LMK if it's any other way around it.

## Related issues

**Checklist**:
- [ ] add related issues
- [ ] update Configuration in README
- [ ] update Changelog in README (major versions)
- [ ] update Upgrading in README (breaking changes)
- [x] ready for review

### CODE REVIEW

**Comment by travisgroth** [comment]
Hi @adinhodovic - can you explain the environment this is intended for?  Your ingress controller doesn't terminate TLS but the load balancer in front of it does?

**Comment by adinhodovic** [comment]
> Hi @adinhodovic - can you explain the environment this is intended for? Your ingress controller doesn't terminate TLS but the load balancer in front of it does?

Yeah exactly, EKS with an ACM cert that the AWS LB uses - usually use letsencrypt + ingress but not for one of the setups I'm working at.

**Comment by travisgroth** [comment]
> Yeah exactly, [EKS with an ACM cert that the AWS LB uses](https://kubernetes.github.io/ingress-nginx/deploy/#tls-termination-in-aws-load-balancer-nlb) - I usually use letsencrypt + ingress but not for one of the setups I'm working at.

Got it.  I think that's a simple enough change.  I think you're going to get a lint error - LGTM once tests are green.

**Review by travisgroth — APPROVED** [review]

**Pull request merged**
